docs/content/troubleshooting/vpn.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Using Docksal with VPNs"
+weight: 2
+---
+
+## Using Docksal with VPNs
+
+Some VPNs are configured to intercept and re-route all traffic (let's call them "greedy"), thus breaking 
+access to the 192.168.64.100 IP that Docksal is using. Attempting to change network routing while connected to
+a greedy VPN will usually result in VPN dropping the connection. Enterprise security at work here.
+
+The one thing VPNs cannot do though, is mess with the hosts loopback interface (127.0.0.1). That's how your 
+traditional web server is not affected by a greedy VPN connection - it just binds to 127.0.0.1:80 (or more 
+likely to 0.0.0.0:80 - port 80 on all network interfaces).
+
+Binding to 0.0.0.0:80 can be achieved with Docksal like this:
+```
+fin config set --global DOCKSAL_VHOST_PROXY_IP=0.0.0.0
+fin system reset vhost-proxy
+```
+You can then access http://localhost (or https://127.0.0.1) and see the "Project missing" page from Docksal's
+vhost-proxy container.
+
+![project missing](/images/vpn-missing.png)
```

Comment:
I'd say, the image can be dropped as it does not present a lot of value and looks awkward as the only image on the VPN setup page

Reply:
Done.